keps/sig-network/4410-k8s-network-interface/README.md

```diff
@@ -214,6 +225,7 @@ and make progress.
 
 1. Any changes to the kube-scheduler 
 2. Any specific implementation other than the reference implementation. However we should ensure the KNI-API is flexible enough to support
+3. Changes to the Pod specification 
```

Comment:
Should we make a caveat, that we want to avoid these changes _UNLESS_ all other options are exhausted?

Reply:
I think that sounds reasonable. We will need a significant amount of evidence to support this change. Looking at the reasoning behind the need for the field and why the existing label selector pattern for services wouldn't work with the network resource as well would need to be looked at.

Reply:
```suggestion
3. Changes to the Pod specification (if we can avoid it: if we see a _very_ strong need then we may need to)
```

Reply:
I went ahead and removed this as a non-goal and moved it the notes/constraints section.